### ISSUE TITLE
Unable to run server mode on local machine

### ISSUE THREAD

**Issue by NileshGule** [groupon/sparklint#64]
I ma trying to run the Sparkling server mode on local machine on MacOSX version 10.12.6. I have installed SBT.After cloning the repo, I tried to sbt run command. I get the attached error related to unresolved dependency. 
![sbt error](https://user-images.githubusercontent.com/1298374/29279676-37153424-814b-11e7-8bb9-1417207bd9f6.png)

**Comment by roboxue** [comment]
I have no idea about this issue, very likely this is environment related issue. `sbt run` on my side as well as on all my linux boxes runs fine.

According to your log, this is maven/ivy2 not able to download a scala library from the remote repository. Please check if you have any enterprise level nexus proxy / network issue....

**Comment by NileshGule** [comment]
I am running the sbt run on my personal Mac. There is no firewall or network issue.

**Comment by roboxue** [comment]
@NileshGule have solved this SBT issue now?

**Comment by NileshGule** [comment]
No, I am still getting the same error. I have Scala 2.10_6 version installed and I cleaned the ivy2 directory and rerun the command but no luck :(
